main.go

```diff
@@ -81,6 +82,21 @@ func archive(path string) error {
 	return cmd.Run()
 }
 
+func checkPrerequisites() error {
+	if _, err := exec.LookPath("oc"); err != nil {
```

Comment:
We could omit this deferring to only call `exec.Command("oc", "whoami").Run()`, but then we'd have to have a longer, more generic error message.